### ISSUE TITLE
Status endpoint needed

### ISSUE THREAD

**Issue by ex3ndr** [jmorganca/ollama#1246]
Hello!

I found a non-urgent issues in the API that makes UX much worse when working with models from web or with remote servers because we can't see current state of a ollama: is it downloading model? did it fail downloading model? is it doing inference? how much RAM/VRAM is used? Also lack of such status endpoint it is not clear what to do if connection was aborted during pull - how to check the status of pull operation? 

Lack of this endpoint ends up in a weird UI in most projects i have seen so far.

**Comment by paulrobello** [comment]
I agree, at the very least an endpoint that provides the same info as:
ollama ps

**Comment by dhiltgen** [comment]
A few of these are covered by #7262

**Comment by MadDenker** [comment]
Agree with this.  The only way I know the local model is working is by checking Task Manager or htop.  I like the simple, clean UI, but some simple status information on the status bar would be very helpful.

**Comment by parawanderer** [comment]
Hi, I had the same interest and just rebased #7262 in this PR: https://github.com/ollama/ollama/pull/18142
Curious if that addresses what other people here were interested in?
